CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # tripwire_agent Cookbook CHANGELOG
 
-## 0.1.3 (2017-10-02)
+## 0.1.4 (2017-10-25)
+
+Resolved nil issue when using the java agent resource
+
+Added new_resource to all properties in axon and java resources for Chef13 support
+
+## 0.1.3 (2017-09-29)
```

Comment:
I think you had a conflict here because I changed the release date to match the actual date I pushed it. You probably need to re-sync your fork with the official one and fix the date.

Just leave the date of the next release TBD until I actually push it.

Reply:
Chef 13 updates needed to happen and was in the same place. It's just a mess if you don't fix it now since the run complains about it (bad customer experience!). 

Thanks for the heads up on the conflict and dates for the Changelog. I wasnt sure what happened or how I had something so different.